contents/blog/the-truth-about-tiger-teams.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: What no one tells you about tiger teams
```

Comment:
If we are making this a newsletter, we might want to title it like:

"Why small teams crush tiger teams" 

"Tiger teams will kill your product, small teams can fix it"